sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -506,6 +506,25 @@ class HiveDDLSuite
     }
   }
 
+  test("desc formatted table for permanent view") {
+    withTable("tbl") {
+      withView("view1") {
+        sql("CREATE TABLE tbl(a int)")
+        sql("CREATE VIEW view1 AS SELECT * FROM tbl")
```

Comment:
Hi, @gatorsmile . 
Looks nice. Could you improve this as a Hive syntax including partition?
```
CREATE VIEW ~~~ PARTITIONED ON (~~)
```
https://cwiki.apache.org/confluence/display/Hive/PartitionedViews